addons/project/static/src/project_sharing/components/chatter/chatter_composer.xml

```diff
@@ -0,0 +1,61 @@
+<templates id="template" xml:space="preserve">
+
+    <!-- Widget PortalComposer (standalone)
+
+        required many options: token, res_model, res_id, ...
+    -->
+    <t t-name="project.ChatterComposer" owl="1">
+        <div t-if="props.allowComposer" class="o_portal_chatter_composer">
+            <t t-if="props.displayComposer">
+                <div t-if="state.displayError" class="alert alert-danger mb8 o_portal_chatter_composer_error" role="alert">
+                    Oops! Something went wrong. Try to reload the page and log in.
+                </div>
+
+                <div class="d-flex">
+                    <img t-if="!props.isUserPublic or props.token"
+                         alt="Avatar"
+                         class="o_portal_chatter_avatar o_object_fit_cover align-self-start"
+                         t-attf-src="/web/image/res.partner/{{ props.partnerId }}/avatar_128"
+                    />
+                    <div class="flex-grow-1">
+                        <div class="o_portal_chatter_composer_input">
+                            <div class="o_portal_chatter_composer_body mb32">
+                                <TextField
+                                    rowCount="4"
+                                    placeholder="'Write a message...'"
+                                    value="state.message"
+                                    update.bind="update"
+                                />
+                                <ChatterAttachmentsViewer
+                                    attachments="state.attachments"
+                                    canDelete="true"
+                                    delete.bind="deleteAttachment"
+                                />
+                                <div class="mt8">
+                                    <button name="send_message" t-ref="sendButton" t-on-click="sendMessage" class="btn btn-primary me-1" type="submit">
```

Comment:
Why not using a state (instead of the t-ref) that you could alter in your code and a `t-att-disabled=[state.zboob]` here?

Reply:
Indeed you are right. :)